wdyr.ts

```diff
@@ -1,9 +1,9 @@
-// Implements Why Did You Render (WDYR) in Dev
-import lodashGet from 'lodash/get';
+/// <reference types="@welldone-software/why-did-you-render" />
+// Implements Why Did You Render (WDYR) in Dev, setup according to: https://github.com/welldone-software/why-did-you-render?tab=readme-ov-file#setup
```

Comment:
```suggestion
import type WhyDidYouRender from '@welldone-software/why-did-you-render';
```

A modern way to do this is just use `import`, it will produce the same result!

```ts
import '@welldone-software/why-did-you-render';
```

But, for this, case let's do what I suggested because we need the `WhyDidYouRender` type for the comment below